openc3-cosmos-init/plugins/packages/openc3-vue-common/src/components/Openc3Screen.vue

```diff
@@ -308,10 +308,13 @@ export default {
     },
     computedStyle() {
       let style = {}
+      // note down what the width was in case it was set to AUTO, because absolute positioning will lose that
+      const origWidth = this.width || this.$refs.bar?.clientWidth
```

Comment:
I was surprised that setting a width and height doesn't actually change the width / height. We set the vcard `min-height` and `min-width` so the only option is to set a screen bigger than the actual layout. I was expecting a scrollable window with a set width and height. Thoughts? This could be another ticket later.